pkg/accounting/rollup/rollup.go

```diff
@@ -66,21 +66,25 @@ func (r *Service) Rollup(ctx context.Context) error {
 	if err != nil {
 		return Error.Wrap(err)
 	}
-	if len(rollupStats) == 0 {
-		r.logger.Info("RollupStats is empty after RollupStorage")
-	}
+
 	err = r.RollupBW(ctx, lastRollup, rollupStats)
 	if err != nil {
 		return Error.Wrap(err)
 	}
+
+	//remove the latest day (which we cannot know is complete), then push to DB
+	latestTally = time.Date(latestTally.Year(), latestTally.Month(), latestTally.Day(), 0, 0, 0, 0, latestTally.Location())
```

Comment:
what does latestTally.Location() return? Does it work ok with UTC?

Reply:
Also we can wrap lines 76 and 77 in a conditional that checks the config for enablement

Reply:
Inside ```RollupStorage``` I made sure to convert latestTally to UTC